### DISCUSSION TITLE
grid - show SQL statement

### DISCUSSION THREAD

**Discussion by teafmartins** [serenity-is/Serenity#6960] [Help]
Hi,
Is there any way, for quick debugging, to show the SQL statement that was executed above the grid?
Thanks

**Comment by luongnv6** [comment]
You can override Before...()
then debug right here
![image](https://github.com/serenity-is/Serenity/assets/21300748/5769ca19-30c3-4f51-849a-703b615d31b5)

**Comment by teafmartins** [comment]
But I need to shoe the query stament in the screen above the grid? Know how to do it?
